eng/Directory.Build.Data.targets

```diff
@@ -19,9 +19,26 @@
       <DirectoryPartofPath>$(PackageRootPath.Replace($(RepoRoot), ''))</DirectoryPartofPath>
       <PackageProjectUrl Condition="Exists('$(PackageRootPath)/README.md')">$([System.UriBuilder]::new($(RepositoryUrl)/blob/$(SourceRevisionId)/$(DirectoryPartofPath)README.md).Uri)</PackageProjectUrl>
       <PackageReleaseNotes Condition="Exists('$(PackageRootPath)/CHANGELOG.md')">$([System.UriBuilder]::new($(RepositoryUrl)/blob/$(SourceRevisionId)/$(DirectoryPartofPath)CHANGELOG.md).Uri)</PackageReleaseNotes>
+      <PackageReleaseNotes Condition="'$(ExtractedReleaseNotes)' != ''">$(ExtractedReleaseNotes)</PackageReleaseNotes>
     </PropertyGroup>
   </Target>
 
+  <Target Name="GetCurrentReleaseNotes" AfterTargets="Restore" BeforeTargets="Build" Condition="'$(SkipDevBuildNumber)' == 'true' and '$(IsShippingClientLibrary)' == 'true'" >
```

Comment:
Do you need the BeforeTargets="Build"? I wouldn't expect that to be necessary.

Reply:
Can we do this only for Pack?

Reply:
It does run only for pack, because it runs before `SetPackageProjectUrlandReleaseNotes` which depends on a target in pack.